api/genomics/src/main/java/org/pmiops/workbench/genomics/RandomizeVcfEntryPoint.java

```diff
@@ -0,0 +1,7 @@
+package org.pmiops.workbench.genomics;
+
+public class RandomizeVcfEntryPoint {
+  public static void main(String[] argv) {
+    new RandomizeVcf().instanceMain(argv);
```

Comment:
opt: personal preference, but I think this could just go into the RandomizeVcf file directly, rather than having the separate entrypoint file. I don't feel strongly.

Reply:
I'll give a try; if it works I'll get rid of these. I _do_ need the combine entry point cause I'm not extending anything from Picard